sentry/templates/configmap-relay.yaml

```diff
@@ -1,6 +1,8 @@
-{{- $redisHost := include "sentry.redis.host" . -}}
-{{- $redisPort := include "sentry.redis.port" . -}}
-{{- $redisPass := include "sentry.redis.password" . -}}
+{{- $redisHost   := include "sentry.redis.host" . -}}
+{{- $redisPort   := include "sentry.redis.port" . -}}
+{{- $redisPass   := include "sentry.redis.password" . -}}
```

Comment:
I see that definition of `sentry.redis.password` haven't changed – how will it pick up `externalRedis.existingSecret`?

Reply:
Unfortunately I could not get redis password in the relay config. But I solved that by setting a redis url. if that env var is set, the password is not needed.

```
  env:
    - name: RELAY_REDIS_URL
      valueFrom:
        secretKeyRef:
          name: sentry-secret
          key: sentry-redis-url
```